src/components/UserConfigurableDataGrid/index.tsx

```diff
@@ -3,7 +3,10 @@ import { DataGridPro, DataGridProProps } from '@mui/x-data-grid-pro';
 import useConfigurableDataGridColumns from './useConfigurableDataGridColumns';
 import { useModelsFromQueryString } from './useModelsFromQueryString';
 
-type UserConfigurableDataGridProps = DataGridProProps & {
+type UserConfigurableDataGridProps = Omit<
```

Comment:
Nice!

Reply:
yay! spent some extra time getting more comfy with the Omit-Pick-Partial-etc, feels well worth it :)